addons/cargo/functions/fnc_deploy.sqf

```diff
@@ -0,0 +1,143 @@
+#include "script_component.hpp"
+/*
+ * Author: Smith
+ * Starts the deploy process for object to unload.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_cargo_fnc_deploy
+ *
+ * Public: No
+ */
+
+disableSerialization;
```

Comment:
No.

Reply:
Do you think that better to have special function for getting selected item?

Reply:
I moved getting a selected item into separate function.